test/extensions/geoip_providers/maxmind/geoip_provider_test.cc

```diff
@@ -265,6 +333,53 @@ TEST_F(GeoipProviderTest, ValidConfigCityMultipleLookups) {
   expectStats("city_db", 2, 2);
 }
 
+TEST_F(GeoipProviderTest, DbReloadedOnMmdbFileUpdate) {
+  const std::string config_yaml = R"EOF(
+    common_provider_config:
+      geo_headers_to_add:
+        country: "x-geo-country"
+        region: "x-geo-region"
+        city: "x-geo-city"
+    city_db_path: {}
+  )EOF";
+  std::string city_db_path = TestEnvironment::substitute(
+      "{{ test_rundir "
+      "}}/test/extensions/geoip_providers/maxmind/test_data/GeoLite2-City-Test.mmdb");
+  std::string reloaded_city_db_path = TestEnvironment::substitute(
+      "{{ test_rundir "
+      "}}/test/extensions/geoip_providers/maxmind/test_data/GeoLite2-City-Test-Updated.mmdb");
+  const std::string formatted_config =
+      fmt::format(config_yaml, TestEnvironment::substitute(city_db_path));
```

Comment:
looks like current tests failed because c++20 expects `config_yaml` to be a constant expression
```
test/extensions/geoip_providers/maxmind/geoip_provider_test.cc:352:19: error: call to consteval function 'fmt::basic_format_string<char, std::string>::basic_format_string<std::string, 0>' is not a constant expression
      fmt::format(config_yaml, TestEnvironment::substitute(city_db_path));
```